### ISSUE TITLE
Make it easier to run on the cloud

### ISSUE THREAD

**Issue by friggog** [microsoft/OpticSim.jl#40]
It would be great if it was easier to run large simulations on the cloud.
It would be pretty easy to set up a custom docker image with Julia installed and ideally a sysimage there too with the OpticSim dependencies in - this would be hugely valuable in itself for cloud execution.

To go even further we could then have a script within the default package which submits runs e.g. to Azure ML using this docker image and whatever command the user wants to execute.
AzureML has a nice thing where it uploads a snapshot of the source directory, so it is possible to modify the source code and run jobs with those modifications.

e.g.
```julia
OpticSim.submit_job_to_AML(,aml_workspace, relpath_to_entry_script, args_for_script)
```

I doubt there is a julia wrapper for AML API, but we could probably call the python API somehow...

Certainly not sure on the details here, but enabling cloud runs seems like something that should be reasonably high priority

**Comment by alfredclwong** [comment]
This ties in nicely with our push towards Pluto notebooks as we won't want to do heavy (or even light) simulations on a web server. I'll look into it when I can.

**Comment by BrianGun** [comment]
@friggog were you going to take a crack at this? Sounds like you have the most knowledge about AzureML.

**Comment by friggog** [comment]
Yeah I’m happy to take a look, at least to get the basics running and then maybe you can add a more comprehensive framework on top of that

**Comment by friggog** [comment]
Resolved by #71
